src/gameData/GameConsts.h

```diff
@@ -24,6 +24,13 @@
 /// Geschwindigkeitsabstufungen - Längen der GFs in ms
 const boost::array<unsigned, 6> SUPPRESS_UNUSED SPEED_GF_LENGTHS = {{80, 60, 50, 40, 30, 1}};
 
+const boost::array<float, 5> ZOOM_FACTORS = { 0.5f, 0.75f, 1.f, 1.25f, 1.5f};
```

Comment:
These are not game constants. I'd suggest a new file either GuiConsts or ZoomConsts, so GUI and Game are not mixed